### PR TITLE
Add support for hiding all fiat values

### PR DESCRIPTION
**Pull request by csillag**

### CODE REVIEW

File: src/app/utils/faucet-links.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const faucetLinks: Partial<Record<Network, Partial<Record<Layer, Partial<Record<
     [Layer.consensus]: { [Ticker.TEST]: testnetFaucetUrl },
     [Layer.emerald]: { [Ticker.TEST]: `${faucetParaTimeBaseUrl}emerald` },
     [Layer.sapphire]: { [Ticker.TEST]: `${faucetParaTimeBaseUrl}sapphire` },
-    [Layer.pontusx]: { [Ticker.TEST]: 'mailto:contact@delta-dao.com?subject=tokens' },
+    [Layer.pontusx]: { [Ticker.EUROe]: 'mailto:contact@delta-dao.com?subject=tokens' },
```
Comment (buberdds) [inline src/app/utils/faucet-links.ts:12]: we should improve subject to "Request test tokens"

File: src/config.ts [hunk 2]
```diff
@@ -197,3 +197,5 @@ export const getTokensForScope = (scope: SearchScope | undefined): NativeTokenIn
 
 export const getFiatCurrencyForScope = (scope: SearchScope | undefined) =>
   (scope ? paraTimesConfig[scope.layer]?.[scope.network]?.fiatCurrency : undefined) ?? 'usd'
+
+export const noFiatValues = process.env.REACT_APP_NO_FIAT_VALUES === 'true'
```
Comment (buberdds) [inline src/config.ts:201]: I think it was already discussed in a prev PR. REACT_APP_SHOW_FIAT_VALUES and than don't need to use logical not in all places.

File: src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx [hunk 3]
```diff
@@ -53,7 +53,7 @@ export const ParaTimeSnapshot: FC<{ scope: SearchScope }> = ({ scope }) => {
           <Nodes scope={scope} />
         </StyledGrid>
         <StyledGrid item xs={22} md={5}>
-          {!mainToken.free && <TokenPriceCard token={mainToken} />}
+          {!noFiatValues && !mainToken.free && <TokenPriceCard token={mainToken} />}
```
Comment (buberdds) [inline src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx:56]: What card we gonna show for PontusX mainnet?

Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx:56]: There is no mainnet yet, only testnet. For testnet, they explicitly asked not to show the token price card, but show the "ask for test tokens" card. For mainnet, we will see if/when we get there.

File: src/app/utils/faucet-links.ts (hunk 1 above)
Comment (csillag) [reply src/app/utils/faucet-links.ts]: Done

File: .env.production [hunk 4]
```diff
@@ -20,3 +20,4 @@ REACT_APP_STAGING_URLS=https://explorer.stg.oasis.io
 REACT_APP_SHOW_BUILD_BANNERS=true
 # REACT_APP_FIXED_NETWORK=testnet
 # REACT_APP_FIXED_LAYER=sapphire
+REACT_APP_HOW_FIAT_VALUES=true
```
Comment (buberdds) [inline .env.production:23]: typo

File: src/global.d.ts [hunk 5]
```diff
@@ -25,6 +25,7 @@ declare global {
       REACT_APP_STAGING_URLS?: string
       REACT_APP_FIXED_NETWORK?: string
       REACT_APP_FIXED_LAYER?: string
+      REACT_APP_NO_FIAT_VALUES?: string
```
Comment (buberdds) [inline src/global.d.ts:28]: wrong var name

File: .env.production (hunk 4 above)
Comment (csillag) [reply .env.production]: fixed

File: src/global.d.ts (hunk 5 above)
Comment (csillag) [reply src/global.d.ts]: fixed

**Review by buberdds — APPROVED** [review]

File: src/app/utils/url.ts [hunk 6]
```diff
@@ -1,4 +1,4 @@
-const validProtocols = ['http:', 'https:', 'ftp:', 'ipfs:', 'data:']
+const validProtocols = ['http:', 'https:', 'ftp:', 'ipfs:', 'data:', 'mailto:']
```
Comment (lukaw3d) [inline src/app/utils/url.ts:1]: This is dangerous https://github.com/oasisprotocol/oasis-wallet-web/issues/228

Comment (csillag) [reply src/app/utils/url.ts:1]: Right, but to frame this properly,  currently the only place we are using mailto links are the faucet links, which are hardwired in the code.

Maybe we should add this as an exception for the faucet link only, and not in the general frunction?

File: src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx [hunk 7]
```diff
@@ -53,7 +53,7 @@ export const ParaTimeSnapshot: FC<{ scope: SearchScope }> = ({ scope }) => {
           <Nodes scope={scope} />
         </StyledGrid>
         <StyledGrid item xs={22} md={5}>
-          {!mainToken.free && <TokenPriceCard token={mainToken} />}
+          {showFiatValues && !mainToken.free && <TokenPriceCard token={mainToken} />}
```
Comment (lukaw3d) [inline src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx:56]: what non-free token are we hiding now?

Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/ParaTimeSnapshot.tsx:56]: The Pontus-X guys don't want to see _any_ fiat values anywhere in their version, not even the price card. So, in their deployment, `showFiatValue` will be set to `false` via `.env`, and so the token price card (for EUROe) will be hidden.

File: src/app/utils/url.ts (hunk 6 above)
Comment (lukaw3d) [reply src/app/utils/url.ts:1]: > Right, but to frame this properly, currently the only place we are using mailto links are the faucet links, which are hardwired in the code.

no, it was ensuring safety of validator logos and NFT links too

Comment (lukaw3d) [reply src/app/utils/url.ts:1]: > Maybe we should add this as an exception for the faucet link only, and not in the general frunction?

sure, we can consider all our external links as safe

**Pull request merged**